scripts/release/release-test.R

```diff
@@ -868,9 +879,9 @@ cli_alert_info(sprintf("Creating project [%s]", omics_project))
 armadillo.create_project(omics_project)
 rda_file_body <- upload_file(rda_dir)
 cli_alert_info(sprintf("Uploading resource file to %s into project [%s]", armadillo_url, omics_project))
-system.time({
-  post_resource_to_api(omics_project, token, auth_type, rda_file_body, "ewas", "gse66351_1.rda")
-})
+# system.time({
```

Comment:
Why did you comment this out?

Reply:
Ah that was a mistake. This step takes a long time to run so when testing the script I commented it out to speed things up - added back in :)